lago.spec.in

```diff
@@ -79,6 +79,7 @@ BuildRequires: python-dulwich
 %endif
 BuildRequires: python-flake8
 BuildRequires: git
+BuildRequires: python-netaddr
```

Comment:
Because of the dynamic generation of the config:
```%{buildroot}/usr/bin/lago --ignore-warnings generate-config > "%{buildroot}/%{_sysconfdir}/lago/lago.conf"```